test/fixtures/krb5kdc-fixture/build.gradle

```diff
@@ -0,0 +1,84 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+apply plugin: 'elasticsearch.build'
+
+def vagrantEnvVars = [
+        'VAGRANT_CWD'           : "${project.projectDir.absolutePath}",
+        'VAGRANT_VAGRANTFILE'   : 'Vagrantfile',
+        'VAGRANT_PROJECT_DIR'   : "${project.projectDir.absolutePath}"
+]
+
+def box = "krb5kdc"
+
+def defaultPrincipals = [ "elasticsearch" ]
+
+task update(type: org.elasticsearch.gradle.vagrant.VagrantCommandTask) {
```

Comment:
These tasks were built out originally to make sure that you could stand up the VM via Gradle and tear it down separate from any other processes. This is essentially similar in spirit to the HDFS Fixture build file that allows a dev to standup the HDFS fixture on its own. I can tear this stuff out if it doesn't add any benefit.